### DISCUSSION TITLE
Pagination Data from external API

### DISCUSSION THREAD

**Discussion by richarddice** [livewire/livewire#4995] [1. Help]
Hi, seeking for help on how to use pagination where the data is come from external API. this is my first time using livewire datatable and pagination thank you. tried searching if there's similar concern but I found nothing. Hoping someone can help me.

**Comment by joshhanley** [comment]
@richarddice you will probably need to create a manual paginator and pass it the data. Should be something in one of the Laravel forums about creating a paginator from API data.

Hope this helps!
